src/components/blog/TextBlock/TextBlock.module.css

```
@@ -0,0 +1,44 @@
+.bleedWrapper {
+  margin: auto;
+  max-width: 630px;
+}
+
+.richText h1,
+h2,
+h3,
+h4,
+h5 {
```

```suggestion
.richText h1,
.richText h2,
.richText h3,
.richText h4,
.richText h5 {
```

You need to prefix each of these with `.richText`, since the comma has lower precedence than all other operators in CSS selectors. Without the additional `.richText` prefixes, this actually ends up applying to _all_ `<h2>`, `<h3>`, etc. elements, even those outside of the `.richText` area.

Same comment goes for all the other selectors in this file that don't have `.richText` before the tag selector.